openfga/v1/openfga_service.proto

```diff
@@ -470,6 +470,8 @@ service OpenFGAService {
   }
 
   rpc ReadTuples(ReadTuplesRequest) returns (ReadTuplesResponse) {
+    option deprecated = true;
```

Comment:
Let's look into why this doesn't change anything in the generated OpenAPI spec. I would expect to see something like

```
/stores/:storeId/read-tuples:
    post:
      deprecated: true
      ...
```
in the OpenAPI spec.

Reply:
Hmm... looks like it is not possible yet: https://github.com/grpc-ecosystem/grpc-gateway/issues/2906

Reply:
Ok, let's explicitly add the `deprecated` annotations to the OpenAPI annotations for the method then.

e.g.

```
  rpc ReadTuples(...)
    option deprecated = true;
    
    option (google.api.http) = {...};

    option (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_operation) = {
      ...
      deprecated: true
```

Reply:
Comment added. Thanks!